### PR TITLE
Exercise: Month Names

### PR DESCRIPTION
**Pull request by artem-konovalov-personal**
Closed [#510](https://github.com/nesterone/sf-eng-init-gl-spring-2016/issues/510)

### CODE REVIEW

**Comment by artem-konovalov-personal** [comment]
I just have added solution with implementing module system using **WebPack**

**Comment by artem-konovalov-personal** [comment]
Now it works without WebPack

File: 01-syntax/09-modules/month-names/month.js [hunk 1]
```diff
@@ -1,3 +1,10 @@
+/* global month */
```
Comment (nesterone) [inline 01-syntax/09-modules/month-names/month.js]: You may remove that file it's useless

**Comment by artem-konovalov-personal** [comment]
I have removed `month.js` file as useless one according to your advice

**Pull request merged**